universal_transfer_operator/docs/universal_transfer_operator/operator.rst

```diff
@@ -0,0 +1,45 @@
+.. _universal_transfer_operator:
+
+========================================================================================================
+:py:mod:`universal_transfer_operator operator <universal_transfer_operator.universal_transfer_operator>`
+========================================================================================================
+
+When to use the ``universal_transfer_operator`` operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The Universal Transfer Operator allows data transfers between any :ref:`dataset`. It copies all the data from the source
+Dataset to the destination Dataset. The DAG author only needs to use the Universal Transfer Operator for all transfers.
+
+This ensures a consistent set of :py:mod:`Data Providers <universal_transfer_operator.data_providers>` that can read
+from and write to Datasets. The Universal Transfer Operator can use any Data Provider as a source or a destination. It
```

Comment:
The first sentence in this para says that it allows transfers between "datasets", in this sentence we are saying it allows transfers between "Data provider" ?